ethclient/ethclient_test.go

```diff
@@ -598,8 +598,9 @@ func testAtFunctions(t *testing.T, client *rpc.Client) {
 	if err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
-	if pending != 1 {
-		t.Fatalf("unexpected pending, wanted 1 got: %v", pending)
+	// After nerfing the pending block, pending transaction count will be 0
```

Comment:
![Screenshot 2023-11-28 at 13-59-11 verb nerf at DuckDuckGo](https://github.com/ethereum/go-ethereum/assets/142290/c56c26d6-66d2-4ff4-9fd9-08c267cc24fe)

Is this what you mean? Or do you mean 'yeet'?

Reply:
I meant nerf as in weaken. The pending block is still constructed, just without any transactions